components/studio/bin/hab-studio.sh

```diff
@@ -844,14 +730,14 @@ info() {
   fi
 
   if [ "${HAB_NOCOLORING:-}" = "true" ]; then
-    printf -- "   ${program:-unknown}: $1\n"
+    echo "   ${program:-unknown}: $1"
   else
     case "${TERM:-}" in
       *term | xterm-* | rxvt | screen | screen-*)
-        printf -- "   \033[1;36m${program:-unknown}: \033[1;37m$1\033[0m\n"
+        echo "   \033[1;36m${program:-unknown}: \033[1;37m$1\033[0m"
```

Comment:
Sorry, this is the more problematic call

Reply:
Ah, good catch. The linter complained about the use of [variables in the format string](https://github.com/koalaman/shellcheck/wiki/SC2059), but we can change this to
```bash
        printf -- "   \033[1;36m%s: \033[1;37m$1\033[0m\n" "${program:-unknown}"
```
While we're here though, should it be printing all the args and not just the first?
```bash
        printf -- "   \033[1;36m%s: \033[1;37m$*\033[0m\n" "${program:-unknown}"
```